bin/diff_files.py

```diff
@@ -21,6 +21,10 @@ def get_type(filename):
     return "dump"
 
 
+# Fields the collector adds when it ingests a crash report
+COLLECTOR_FIELDS = ["timestamp", "submitted_timestamp", "collector_notes", "payload"]
```

Comment:
`collector_notes` was added ages ago to capture notes that the collector changed the raw crash data in some way--like removing items from it.

`payload` was also added ages ago to log what shape the crash annotations had in the payload--was it a bunch of key/val pairs in multipart/form-data? or was it a JSON value?